src/App.js

```diff
@@ -13,35 +13,84 @@ import VisitPage from './containers/VisitPage';
 import Footer from './components/Footer';
 import Header from './components/Header';
 
+// LOCALIZATION
+import { getTextForLanguage } from './utils/getTextForLanguage';
+
 
 // Change HashRouter tags below to Router tags to turn off hash routing; only used to be compatible with GitHub Pages
-const App = () => {return (
-  <div id='App'>
-    <HashRouter getUserConfirmation={ getUserConfirmation }>
-      <div id='HashRouter'>
-        <Route
-          path="/:rest+"
-          component={ Header } />
-        <Route
-          exact
-          path="/"
-          component={ HomePage } />
-        <Route
-          path="/about"
-          component={ AboutPage } />
-        <Route
-          path="/visit/:clientId/:visitId"
-          component={ VisitPage } />
-        <Route
-          path="/visit/load"
-          component={ VisitPage } />
-        <Route
-          path="/load"
-          component={ VisitPage } />
+class App extends Component {
+  constructor (props) {
+    super(props);
+    this.state = { language: 'en', snippets: getTextForLanguage('en') };
+  }
+
+  setLanguage = (inputProps) => {
+    var snippets = getTextForLanguage(inputProps.value);
+    this.setState({ snippets: snippets });
```

Comment:
Should this function also set `this.state.language`?

Reply:
Good point!